contracts/shared/helpers/Create2Fixture.ts

```diff
@@ -83,7 +83,19 @@ export default class Create2Fixture {
 
     // If contract doesn't exist at expected address, deploy it there
     if ((await ethers.provider.getCode(contractAddress)) === "0x") {
-      await (await create2Deployer.deploy(salt, initCode)).wait();
+      const receipt = await (
+        await create2Deployer.deploy(salt, initCode)
+      ).wait();
+
+      console.log(
+        [
+          `Deployed ${contractName}`,
+          `using ${receipt.gasUsed.toString()} gas:`,
+          contractAddress,
+        ].join(" "),
+      );
```

Comment:
You should be able to pass this in as [one formatted string or as a comma separated list](https://developer.mozilla.org/en-US/docs/Web/API/Console/log) and let the CLI output format it depending on terminal width. Reason you wanted to use this specific format?